charts/clusterapi-resources/templates/MachineDeployment.yaml

```diff
@@ -13,6 +13,10 @@ spec:
   replicas: {{ $md.replicas }}
   selector:
     matchLabels: {}
+{{- with $md.strategy }}
+  strategy: 
+  {{- toYaml . | nindent 4 }}
```

Comment:
```suggestion
  {{- toYaml . | indent 4 }}
```

?

Reply:
nindent is correct. It has to do with keeping the indentation consistent on the first line and the subsequent lines

Reply:
https://helm-playground.com/#t=LQhQFMA8EMFsAcA24BcACA3h4aB0A1aRAV3AGdcBLAF3FjLQB81qB7ATTkSbQDtLeAE3C9qaAExoAvlNC0y1FKCw4A7jQAWeQiXK55YmaDSYMLDlzw9%2BQkWPFGVaEYOlSgA&v=LQhQEsBcFMFsGcBcACA2gIgGbgE70ugDTLrzQDGA9gHYAm6AuqDPoqMsgO6WcoCMAJgDM7ZFUoAbFEIF8gA